core/rawdb/freezer_batch.go

```diff
@@ -20,9 +20,11 @@ import (
 	"fmt"
 	"sync/atomic"
 
+	"github.com/golang/snappy"
```

Comment:
In our codebase, all the external library imports are in the same block. I would prefer the previous one to align the code style.

Reply:
Yeah my IDE is a bit too eager. I'll fix that.

Reply:
Done